scripts/serverless.js

```diff
@@ -46,8 +52,33 @@ process.once('uncaughtException', (error) =>
     serverless,
     hasTelemetryBeenReported,
     commandUsage,
-  })
-);
+  });
+});
+
+process.on('SIGINT', () => {
+  clearTimeout(keepAliveTimer);
+  if (
+    commandSchema &&
+    !hasTelemetryBeenReported &&
+    !isTelemetryDisabled &&
+    (serverless ? serverless.isTelemetryReportedExternally : true)
+  ) {
+    const telemetryPayload = generateTelemetryPayload({
+      command,
+      options,
+      commandSchema,
+      serviceDir,
+      configuration,
+      serverless,
+      commandUsage,
+    });
+    const failureReason = {
+      kind: 'interrupt',
+    };
+    storeTelemetryLocally({ ...telemetryPayload, failureReason, outcome: 'interrupted' });
```

Comment:
Also I think we need to follow same _grammar_ as with other outcome values.

We have `success`, `failure`, so I believe here should be `interruption` (?)

Reply:
I just checked in dictionory, and best matching noun seems `interrupt` https://www.merriam-webster.com/dictionary/interrupt

Reply:
Great suggestion :+1: